test/unit/start_node_presenter_test.rb

```diff
@@ -56,5 +56,17 @@ class StartNodePresenterTest < ActiveSupport::TestCase
 
       assert_equal "Start now", @presenter.start_button_text
     end
+
+    test "#post_body_header returns custom text when supplied" do
+      @renderer.stubs(:content_for).with(:post_body_header).returns("post-body-header-text")
+
+      assert_equal "post-body-header-text", @presenter.post_body_header
+    end
+
+    test '#post_body_header returns "Before you start" when there is no custom header text' do
+      @renderer.stubs(:content_for).with(:post_body_header).returns("Before you start")
```

Comment:
Should "Before you start" in this test not be `nil`?

Reply:
Just to replicate what would happen when `:post_body_heaader` isn't defined

Reply:
Yes you are absolutely right, thanks Jon 👍